scholia/scrape/ceurws.py

```diff
@@ -405,4 +405,4 @@ def main():
 
 
 if __name__ == "__main__":
-    main()
+    main()
```

Comment:
I do not understand this change. Are you removing the newline?

Reply:
And why do you fix something that is not related to the issue?

Reply:
my bad ceuws.py file should not be changed.

Reply:
Ok. Fine